rllib/env/tests/test_single_agent_gym_env_runner.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+
+import ray
+from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
+from ray.rllib.env.testing.single_agent_gym_env_runner import SingleAgentGymEnvRunner
+
+
+class TestSingleAgentGymEnvRunner(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        ray.init()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        ray.shutdown()
+
+    def test_sample(self):
+        config = (
+            AlgorithmConfig().environment("CartPole-v1")
+            # Vectorize x2 and by default, rollout 64 timesteps per individual env.
+            .rollouts(num_envs_per_worker=2, rollout_fragment_length=64)
+        )
+        env_runner = SingleAgentGymEnvRunner(config=config)
+
+        # Sample 10 episodes 10 times.
```

Comment:
we currently don't support sampling based on `num_episodes` right?

Reply:
Only vie the `config.batch_mode="complete_episodes"` but then you cannot specify the exact number of episodes to sample, correct? UNLESS you are in the eval worker world, where you have the additional settings: `evaluation_duration` (<- num episodes) and `evaluation_duration_unit="episode"`